rules/gsuite_activityevent_rules/google_workspace_admin_custom_role.py

```diff
@@ -0,0 +1,19 @@
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    # Create Alert if there is a custom role created under delegated admin settings
+    return (
+        event.get("type", "") == "DELEGATED_ADMIN_SETTINGS"
+        and event.get("name", "") == "CREATE_ROLE"
+    )
+
+
+def title(event):
+    # (Optional) Return a string which will be shown as the alert title.
+    # If no 'dedup' function is defined, the return value of this method
+    # will act as deduplication string.
+    return (
+        f"Google Workspace Administrator "
+        f"[{event.get('actor',{}).get('email','no-email-provided')}] "
```

Comment:
@edyesed - whats our best practice around non-existing values? I've seen <> used a few times?

Reply:
it's a bit in flux, recent practice has been `<NO_WHATEVER_THING>` with the `<>` and all caps.

Reply:
just made this change!